packages/domain/lib/repository/trakt_repository.dart

```diff
@@ -7,4 +8,6 @@ abstract class TraktRepository {
   Future<GetDataResponse> getDataAnticipated({int? itemCount});
 
   Future<PeopleResponse> getCast({String? movieId});
+
+  Future<List<Comments>> getComments({String? movieId});
```

Comment:
```suggestion
  Future<PeopleResponse> getCast({required String movieId});

  Future<List<Comments>> getComments({required String movieId});
```

Reply:
corrected

Reply:
I don't see it(

Reply:
<img width="573" alt="Screenshot 2022-10-28 at 15 48 56" src="https://user-images.githubusercontent.com/111364147/198590487-2d38dec3-0615-4e06-a4c1-3474050ab3bd.png">

Reply:
change also in the implementation